ui/pages/confirmations/components/confirm/info/shield-subscription-approve/estimated-changes.tsx

```diff
@@ -0,0 +1,51 @@
+import { Box, BoxFlexDirection } from '@metamask/design-system-react';
+import { NameType } from '@metamask/name-controller';
+import { Hex } from '@metamask/utils';
+import React from 'react';
+import { ConfirmInfoRow } from '../../../../../../components/app/confirm/info/row';
+import { ConfirmInfoSection } from '../../../../../../components/app/confirm/info/row/section';
+import Name from '../../../../../../components/app/name';
+import { TextColor } from '../../../../../../helpers/constants/design-system';
+import { useI18nContext } from '../../../../../../hooks/useI18nContext';
+
+export const EstimatedChanges = ({
+  approvalAmount,
+  tokenAddress,
+  chainId,
+}: {
+  approvalAmount: string;
+  tokenAddress: Hex;
+  chainId: Hex;
+}) => {
+  const t = useI18nContext();
+
+  const isMonthlySubscription = approvalAmount === '96';
```

Comment:
we will use data from subscription controller state in a subsequent pr here @tuna1207